README.md

```diff
@@ -1,26 +1,242 @@
 # yarpc-go [![GoDoc][doc-img]][doc] [![Build Status][ci-img]][ci] [![Coverage Status][cov-img]][cov]
 
-A transport and encoding agnostic RPC framework.
+With hundreds to thousands of services communicating with RPC, transport
+protocols (like HTTP and [TChannel][]), encoding protocols (like JSON or
+Thrift), and peer choosers are the concepts that vary.
+Separating these concerns allows services to change transport protocols without
```

Comment:
The terminology here is odd - suggest either just "transports" or "wire protocols" - these are mostly interchangeable, with transports implying a accompanying library that *implements* the wire protocol. Perhaps we should put this in a terminology doc.